.github/workflows/docker-image.yml

```diff
@@ -7,6 +7,7 @@ on:
     tags:
       - 'v*'
   workflow_dispatch:
+  pull_request:
```

Comment:
This was run twice in fact. Let's divide into two jobs: 
* build: runs CI only (ends after current Build step) 
* app-push: runs the rest of the steps in separate job only on on requires branches changes (master and en)